experiments/customui/parent.js

```diff
@@ -285,18 +288,37 @@ var ex_customui = class extends ExtensionCommon.ExtensionAPI {
       }
     }
 
+    // Set all dimensions (width, height and display) of a generic rectangular WebExtension frame
     const setWebextFrameSizesForBox = function(frame, options, isDynamicWidth, width, isDynamicHeight, height) {
       setWebextFrameDimension(frame, options, "width", isDynamicWidth, width);
       setWebextFrameDimension(frame, options, "height", isDynamicHeight, height);
       frame.style.display = options.hidden ? "none" : "block";
     }
+
+    // Enables dynamic width and fixed heigth for a WebExtension frame
+    const setWebextFrameSizesForSidebar = function(frame, options) {
+      //TODO: set maximum width in mailTab to:
+      // width of messengerBox - min-width of folderPaneBox
+      //  - minimal width of messagesBox, coded to 199px (?)
+      //  - width of splitters
+      //  - some small reserve
+      // Probably by adding it as a parameter and add 2 checks in setWebextFrameDynamicDimension
+      //  the function used by CustomUILocalOptionsListener should be the same as setWebextFrameDynamicDimension
+      //  so there is only need for 1 check and 1 place where the dimension is set
```

Comment:
Adding `TODO` marks to a commit is dangerous, as that is usually code for 'never do'. Thunderbird is full of those, many more than a decade old ;)

Reply:
Indeed, in this case it stands for: good idea, but I will probably never do it unless required some day; this is a proposition in pseudo-code.
Additionally creating an issue is probably a good idea.